prometheus-to-sd/go.mod

```diff
@@ -6,9 +6,9 @@ require (
 	cloud.google.com/go/compute/metadata v0.4.0
 	cloud.google.com/go/monitoring v1.20.2
 	github.com/golang/glog v1.2.1
-	github.com/prometheus/client_golang v1.19.1
+	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/client_model v0.6.1
-	github.com/prometheus/common v0.55.0
+	github.com/prometheus/common v0.37.1
```

Comment:
is this a bug in dependabot? downgrading dependencies that should not be touched by `go get golang.org/x/crypto@v0.31.0`